data-pipeline/docker-compose.yml

```diff
@@ -1,14 +1,14 @@
 version: '3'
 services:
   zookeeper:
-    image: confluentinc/cp-zookeeper:5.1.2
+    image: confluentinc/cp-zookeeper:5.1.1
```

Comment:
out of curiosity, why the downgrade?

Reply:
I decided to follow Kafka apis [versions](https://github.com/sysco-middleware/kafka-testing/blob/master/pom.xml#L26). I plan to update all dependencies next PR